Gordon360/Services/AccountService.cs

```diff
@@ -80,6 +87,167 @@ public AccountViewModel GetAccountByUsername(string username)
         return account;
     }
 
+    /// <summary>Indicates whether a user making a request is authorized to see
+    /// profile information for students.</summary>
+    /// <param name="viewerGroups">The authentication groups associated with the 
+    /// user making the request.</param>
+    /// <returns>True if the user making the request is authorized to see
+    /// profile information for students, and false otherwise.</returns>
+    public bool CanISeeStudents(IEnumerable<AuthGroup> viewerGroups)
+    {
+        if (viewerGroups.Contains(AuthGroup.SiteAdmin) ||
+            viewerGroups.Contains(AuthGroup.Police) ||
+            viewerGroups.Contains(AuthGroup.FacStaff) ||
+            viewerGroups.Contains(AuthGroup.Student))
+        {
+            //TODO: take "KeepPrivate" into account, to enforce FERPA restrictions
```

Comment:
I don't think you need to check FERPA restrictions here because you're just determining if you can see students in general.

Reply:
Good point.   TODOs in 2 places were obsolete.